dockers/docker-sonic-mgmt/Dockerfile.j2

```diff
@@ -36,7 +36,8 @@ RUN apt-get update && apt-get install -y apt-transport-https \
                                          sudo \
                                          tcpdump \
                                          telnet \
-                                         vim
+                                         vim \
+                                         unzip
```

Comment:
The unzip is no use in the latest commit, it can be removed, right?

Reply:
Done